Digital Shadows/README.md

```diff
@@ -0,0 +1,5 @@
+# Digital Shadows SearchLight
+
+## Description
+
+Your security team is busy. Digital Shadows SearchLight removes the noise of Threat Intelligence, making it easier to identify what is important so you can make better decisions faster.
```

Comment:
Include an additional sentence to highlight the benefits of enabling this parser